src/containers/AdminContainer/AdminContainer.js

```diff
@@ -19,6 +18,50 @@ class AdminContainer extends React.Component {
         updateEmergency: false,
     }
 
+    static getDerivedStateFromProps(props, state) {
+        const userID = props.match.params.id.split(':')[1]
+        const newState = {
+            ...state,
+            userID: userID,
+        }
+        return newState
+    }
+
+    componentDidMount() {
+        getUser(this.state.userID)
+        .then(response => {
+            if (response['error']) {
```

Comment:
In `getUser`, if there's an `error` property in `response`, it should throw so you can handle the error in the catch instead.

Reply:
there shouldnt be anymore since fixing the backend will not send a successful error